NewForce-Cohort-4/react-nutshell-house-baboonpaw
# Modifying New Articles

# Description
Added the ability to edit articles.
Fixes # (issue) There was no edit button previously on articles.
## Type of change
# Testing Instructions

- You should log in with a registered email.
- Click on articles tab.
- Click New Article button.
- Fill out form.
- Click Add Article.
- Click Edit Article.
- Modify inputs of the form.
- Click Save Article.
- View the Modified Form.

# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
